setup.py

```diff
@@ -58,6 +58,7 @@
         'Programming Language :: Python :: 3.3',
```

Comment:
Deluge is now for Python 3.5+, older versions are unsupported.

Reply:
Will do.